samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/.gitignore

```diff
@@ -0,0 +1,168 @@
+
```

Comment:
Could you please update this .gitignore file to include this additional lines related to powershell/.NET?

```
# Exclude packaged modules
*.zip

# Exclude .NET assemblies from source
*.dll
```